recipes/genera/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = '1.4.2' %}
+
+package:
+   name: genera
+   version: {{ version|replace('-', '_') }}
+
+source:
+  - url: https://github.com/josuebarrera/GenEra/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: acc49698b68151550b825c73f1f03dd268da2c73ecc7ac4c55eccb8bec5b1a54
+  - url: https://github.com/caraweisman/abSENSE/archive/refs/tags/v1.0.0.tar.gz
+    sha256: a99090c10a66afae1fbad38d79d228199e9fbdae5173e71ad5d84b4c958dda54
+    folder: abSENSE
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('genera', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python >=3.8,<3.9
+  run:
+    - python >=3.8,<3.9
+    - diamond >=2.1.10
+    - mmseqs2 =14.7e284
+    - foldseek =3.915ef7d
+    - scipy =1.7.3
+    - r-seqinr =4.2_16
+    - r-optparse =1.7.3
+    - r-bio3d =2.4_3
+    - r-tidyverse =1.3.2
+    - r-phytools >=0.6_99
+    - orthofinder =2.5.5
+    - ncbitax2lin =2.3.2
```

Comment:
I agree with the bot here. Are the exact specifications really necessary?

The `run_exports` of other Bioconda packages will add upper bounds that help constrain co-installation with too new versions in the future.

Also, [Python 3.8 has reached EOL](https://devguide.python.org/versions/#python-release-cycle), so Python 3.9 would make more sense as a minimum.

The `pip` and `setuptools` suggested by the bot can be ignored - they aren't relevant since this only copies scripts.